omero/sysadmins/import-scenarios.txt

```diff
@@ -178,6 +180,8 @@ In-place import
 The following sections outline :ref:`in-place <inplace_import>` based
 scenarios to help you judge if the functionality may be useful for you.
 
+* :doc:`/sysadmins/in-place-import`
```

Comment:
in short order, here are two links to the more or less same place (the in-place import doc). Maybe remove the first one (:ref: `in-place <inplace_import>` and add ``See also....`` in front of the second one